styles/styles.css

```diff
@@ -36,79 +97,79 @@ body {
     height: 40px;
 }
 
-.top-section {
+.top {
     min-height: 100vh;
     background-image: url(../images/background-castle.png);
     background-repeat: no-repeat;
     background-size: cover;
     color: #FFFFFF;
 }
 
+.wrapper--top {
+    padding-top: 50px;
+}
+
+
 .nav {
     height: 46px;
-    padding: 51px 104px 0;
     margin-bottom: 196px;
```

Comment:
It's would be better to replace this margin with an accrual that is a multiple of 5

Reply:
also fixed